tools/Makefile.am

```diff
@@ -28,6 +28,7 @@
 
 include $(abs_top_nlbuild_autotools_dir)/automake/pre.am
 
+if OPENTHREAD_BUILD_TOOLS
```

Comment:
I think we still want to include these directories in `DIST_SUBDIRS` even when `--disable-tools` is set.  Instead, we should wrap the automake rules that involve building in each tool-specific `Makefile.am`.